data/human/culture conversations.txt

```diff
@@ -524,3 +524,65 @@ mission "News Reporter - Paradise"
 			`	"We are fortunate that many people feel this way! Thank you for your answers, and for taking the time to do this interview!" she says.`
 			`	The drones dart back to follow her as she continues through the spaceport.`
 				decline
+
+mission "Scientists at the Pub after a Conference"
+	minor
+	source
+		government "Republic"
+		attributes "urban"
+		not attributes "station"
+	to offer
+		random < 1
+	on offer
+		conversation
+			`After a long journey and a longer-feeling landing procedure, you notice a group of people enter the pub where you're sitting. They stream in from the conference complex across the street. They vary in age, dress, and regional accent, but most have an identical badge dangling from a lanyard around their necks. You notice one that's been dropped on the floor.`
+			`The badge reads "Attendee - 35th Annual Pan-galactic Conference on Interstellar Microbial Genomics and Population Dynamics".`
+			`The group takes their seats a long table near to where you are sitting.`
+			choice
+				`(Order a meal and a drink for yourself.)`
+					goto "overheard scientists 1" 
+				`(Find somewhere quieter to spend your limited time planetside)`
+					decline
+			label "overheard scientists 1"
+			`Fragments of conversation make their way to you through the din of the pub:` 
+			` `
+			`"I'm so glad to have my presentation over with. I was so worried about having to present on the first day, but now I can finally just relax and actually listen to the other talks!"`
+			`"Oh definitely! My talk isn't until the third day. I probably shouldn't even be here - I should really be rehearsing." (She orders another drink anyway)`
+			`---`
+			`"Is there some law I don't know about that requires every 'question' to start with 'I guess this isn't so much a question as a comment...'?"`
+			`---`
+			`"I don't think the nucleotide evidence supports their hypothesis at all! Like, I know the proteomic data could be used to support their conclusion, but they're ignoring synonymous mutations!"`
```

Comment:
Avoid using border lines - they look bad in the ES engine. Instead use descriptions to mark the passage of time. Also avoid using bracketed text outside of action choices (or side comments).